zebra/zebra_vrf.c

```diff
@@ -316,6 +316,13 @@ struct route_table *zebra_vrf_get_table_with_table_id(afi_t afi, safi_t safi,
 	struct other_route_table *otable;
 	struct route_table *table;
 
+	/* VRF config could be deleted while some route messages for the VRF were
```

Comment:
This is a little specific - there are several paths that call this api, not just the one in zebra_rib.c that you were concerned about. How about something more generic, like "vrf_id may be stale" ?

Reply:
thanks for the suggestion